src/lib/orionld/serviceRoutines/orionldPostEntity.cpp

```diff
@@ -514,6 +514,13 @@ bool kjTreeMergeAddNewAttrsOverwriteExisting(KjNode* sourceTree, KjNode* modTree
     //     - add modAttrP to sourceTree
     //     - add slot in "attrNames"
     //
+    if(modAttrP->type != KjObject) 
```

Comment:
The function is called "objectToValue", which implies it should only be called for JSON objects.
Not sure it's a good idea to make the check for object inside the function.
Perhaps better (faster in run-time) to let that responsibility to the caller.
If we do this, a comment in the function header should be added, saying that "It is the responsibility of the caller to make sure that the 'attrP' KjNode pointer is a KJObject"